### PR TITLE
some fixes for pycodestyle E275

### PR DESCRIPTION
**Pull request by fchapoton**
### :books: Description

This fixes some E275 warnings (E275 missing whitespace after keyword) plus a few other things

outside of `graphs`, `rings`, `misc`, `coding`  and `combinat` for now

### :memo: Checklist

- [ ] The title is concise, informative, and self-explanatory.
- [ ] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/geometry/polyhedron/parent.py [hunk 1]
```diff
@@ -400,10 +400,10 @@ def some_elements(self):
         points = []
         R = self.base_ring()
         for i in range(self.ambient_dim() + 5):
-            points.append([R(i*j^2) for j in range(self.ambient_dim())])
+            points.append([R(i * j ^ 2) for j in range(self.ambient_dim())])
```
Comment (mkoeppe) [inline src/sage/geometry/polyhedron/parent.py]: This spacing is a bit much for my taste. Wouldn't `i * j^2` conform to the spec too?

Comment (fchapoton) [reply src/sage/geometry/polyhedron/parent.py]: I am afraid not. Also, I am not sure if XOR is really the intended thing here. changing it to ` **`  breaks some doctests though

Comment (mkoeppe) [reply src/sage/geometry/polyhedron/parent.py]: Oh, right, this is XOR there. So spacing that agrees with precedence would be `i*j ^ 2`

Comment (mkoeppe) [reply src/sage/geometry/polyhedron/parent.py]: (This expression has been there since the very beginning; 7e02dce3d6a)

Comment (mkoeppe) [reply src/sage/geometry/polyhedron/parent.py]: https://peps.python.org/pep-0008/#other-recommendations
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."
```
# Correct:
[...]
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)
# Wrong:
[...]
x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```

**Comment by fchapoton** [comment]
voilà

**Review by mkoeppe — APPROVED** [review]
Thank you, looking good.

**Pull request merged**